retux/client/resources/sticker.py

```diff
@@ -0,0 +1,222 @@
+    APNG = 2
+    """A sticker formatted with the `.apng` format."""
+    LOTTIE = 3
+    """A sticker formatted with the `.lottie` format."""
+
+
+@define(kwargs_only=True)
+class Sticker(Object):
+    """
+    Represents a sticker from Discord.
+
+    Attributes
+    ----------
+    id : `Snowflake`
+        The ID of the sticker.
+    pack_id : `Snowflake`, optional
+        The ID of the pack containing the sticker.
+
+        Only standard stickers owned by Discord
+        can be part of a sticker pack.
+    name : `str`
+        The name of the sticker.
+    description : `str`, optional
+        The description of the sticker.
+    tags : `str`
+        The tags or emoji equivelent of a sticker.
+
+        If the sticker is a standard sticker, this
+        field will be a comma seperated list of 
+        keywords. If the sticker is guild owned, it
+        will be the name of an emoji.
+    asset : `str`
+        An empty string previously for the sticker asset hash.
+    type : `StickerType`
+        The type of the sticker.
+    format_type : `StickerFormatType`
+        The formatting type that the sticker uses.
+    available : `bool`, optinal
+        Whether or not the sticker is availible.
+
+        May be false if the guild that owns it
+        has lost a level of boosting.
+    guild_id : `Snowflake`, optional
+        The ID of the guild that owns the sticker.
+
+        Will only be set to None if the sticker is
+        owned by Discord.
+    user : `User`, optional
+        The user that uploaded the sticker.
+
+        Will only be set to None if the sticker is
+        owned by Discord.
+    sort_value : `int`, optional
+        The sticker's order within its sticker pack.
+
+        Will only be availible if the sticker is
+        part of a pack and owned by Discord.
+    """
+    id: str | Snowflake = field(converter=Snowflake)
+    """The ID of the sticker."""
+    pack_id: str | Snowflake | None = field(converter=optional_c(Snowflake), default=None)
+    """
+    The ID of the pack containing the sticker.
+    
+    Only standard stickers owned by Discord
+    can be part of a sticker pack.
+    """
+    name: str = field()
+    """The name of the sticker."""
+    description: str | None = field(default=None)
+    """The description of the sticker."""
+    tags: str = field()
+    """
+    The tags or emoji equivelent of a sticker.
+    
+    If the sticker is a standard sticker, this
+    field will be a comma seperated list of 
+    keywords. If the sticker is guild owned, it
+    will be the name of an emoji.
+    """
+    asset: str = field(default="")
+    """An empty string previously for the sticker asset hash."""
+    type: int | StickerType = field(converter=StickerType)
+    """The type of the sticker."""
+    format_type: int | StickerFormatType = field(converter=StickerFormatType)
+    """The formatting type that the sticker uses."""
+    available: bool | None = field(default=None)
```

Comment:
```suggestion
    available: bool = field(default=True)
```

I believe reading from the docs they're imposing optional as in identity and not value relation. If the documentation from their website is proper, then I presume it's.. `True` by default?